bumble/l2cap.py

```diff
@@ -2078,7 +2078,8 @@ async def connect(self, connection: Connection, psm: int) -> Channel:
         # Connect
         try:
             await channel.connect()
-        except Exception:
+        except Exception as e:
```

Comment:
maybe use  `try: xxx finally: del yyy` ?

Reply:
moving del to finally would make it wrong, right?
we only want to delete when exception happens?

Reply:
@benquike No you could move the delete to finally 👌

Reply:
I miss-read it!